notebooks/explain-multiclass-classification-local.ipynb

```diff
@@ -0,0 +1,384 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright (c) Microsoft Corporation. All rights reserved.\n",
+    "\n",
+    "Licensed under the MIT License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Explain multiclass classification model's predictions\n",
+    "_**This notebook showcases how to use the interpret-community repo to help interpret and visualize predictions of a multiclass classification model.**_\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Table of Contents\n",
+    "\n",
+    "1. [Introduction](#Introduction)\n",
+    "1. [Project](#Project)\n",
+    "1. [Setup](#Setup)\n",
+    "1. [Run model explainer locally at training time](#Explain)\n",
+    "    1. Train a multiclass classification model\n",
+    "    1. Explain the model\n",
+    "        1. Generate global explanations\n",
+    "        1. Generate local explanations\n",
+    "1. [Visualize results](#Visualize)\n",
+    "1. [Next steps](#Next)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='Introduction'></a>\n",
+    "## 1. Introduction\n",
+    "\n",
+    "This notebook illustrates how to use interpret-community locally to help interpret predictions of a multiclass classification model at training time.\n",
+    "It demonstrates the API calls needed to obtain the global and local interpretations along with an interactive visualization dashboard for discovering patterns in data and explanations.\n",
+    "\n",
+    "Three tabular data explainers are demonstrated: \n",
+    "- TabularExplainer (SHAP)\n",
+    "- MimicExplainer (global surrogate)\n",
+    "- PFIExplainer\n",
+    "\n",
+    "| ![Interpretability Toolkit Architecture](./img/interpretability-architecture.png) |\n",
+    "|:--:|\n",
+    "| *Interpretability Toolkit Architecture* |\n",
+    "\n",
+    "<a id='Project'></a>       \n",
+    "## 2. Project\n",
+    "\n",
+    "The goal of this project is to classify Iris flowers with scikit-learn and run a model interpreter locally.\n",
+    "\n",
+    "1. Train a SVM classification model using Scikit-learn\n",
+    "2. Run 'explain_model' globally and locally with full dataset in local mode, which doesn't contact any Azure services.\n",
```

Comment:
Generally same comments as the other notebook, but in particular I want to flag this mention of Azure